testsupport/tiers/checks.go

```diff
@@ -1786,6 +1788,47 @@ func namespaceManagerSaEditRoleBinding() namespaceObjectsCheck {
 	}
 }
 
+func namespaceManagerSaAdditionalArgocdReadRoleBinding() namespaceObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, memberAwait *wait.MemberAwaitility, owner string) {
+		rb, err := memberAwait.WaitForRoleBinding(t, ns, toolchainv1alpha1.AdminServiceAccountName, toolchainLabelsWaitCriterion(owner)...)
+		require.NoError(t, err)
+		assert.Len(t, rb.Subjects, 1)
+		assert.Equal(t, "ServiceAccount", rb.Subjects[0].Kind)
+		assert.Equal(t, toolchainv1alpha1.AdminServiceAccountName, rb.Subjects[0].Name)
+		assert.Equal(t, "additional-argocd-read", rb.RoleRef.Name)
```

Comment:
Need help fixing this [error I'm seeing on CI](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/codeready-toolchain_host-operator/829/pull-ci-codeready-toolchain-host-operator-master-e2e/1673662020840329216#1:build-log.txt%3A5758):
```
expected: "additional-argocd-read"
actual  : "edit"
```
So, there are now 2 RoleBindings for "namespace-manager", and the `rb.Subjects[0]` here contains the first (already-existing) RoleBinding, so assertions fail because it gets compared to the latest RoleBinding data in this test.

Should I use `rb.Subjects[1]` to do assertions here? But then I'm curious, what's the order in which RoleBindings gets filled in `rb.Subjects[]` array?

Reply:
see https://github.com/codeready-toolchain/toolchain-e2e/pull/754/files#r1243781568